shared/protocol/src/main/java/io/pravega/shared/segment/StreamSegmentNameUtils.java

```diff
@@ -84,6 +86,13 @@
     private static final String TABLES = "_tables";
     private static final String MARK = "_MARK";
 
+    /**
+     * This is used to parse the segment name.
+     */
+    static final Pattern SEGMENT_TAGS_PATTERN =
```

Comment:
Please add `@VisibleForTesting` annotation since the only reason for this not being private is because it's used in unit tests.